packages/google_maps_flutter/google_maps_flutter_web/example/web/index.html

```diff
@@ -5,7 +5,7 @@
 <html>
     <head>
         <title>Browser Tests</title>
-        <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=geometry"></script>
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAa9cRBkhuxGq3Xw3HPz8SPwaVOhRmm7kk&libraries=geometry"></script>
```

Comment:
Could you add a comment here about the source so that if we have issues with it someday in the future someone will know where to go?

Reply:
Absolutely.